### PR TITLE
feat: added testing for most classes

### PR DESCRIPTION
**Pull request by kevinglezgmz**

### CODE REVIEW

File: src/test/java/kevinglezgmz/chess/gameEngine/GameEngineTest.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class GameEngineTest {
+
+  @Test
+  public void shouldAnswerWithTrue() {
+    assertTrue(true);
```
Comment (AlexisGlez) [inline src/test/java/kevinglezgmz/chess/gameEngine/GameEngineTest.java:11]: ? 😂

File: target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst [hunk 2]
```diff
@@ -0,0 +1,9 @@
+c:\Users\Kevin Gonzalez\Desktop\Coding examples for scavan - WEEK 3\chess\src\main\java\kevinglezgmz\chess\App.java
```
Comment (AlexisGlez) [inline target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst:1]: What is "scavan" doing in here? 😂

File: target/surefire-reports/TEST-kevinglezgmz.chess.AppTest.xml [hunk 3]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AlexisGlez) [inline target/surefire-reports/TEST-kevinglezgmz.chess.AppTest.xml:1]: I don't know think this files should be tracked.

The ones inside "target" folder, they seem to be auto generated. Auto generated files must not be included in Git

File: README.md [hunk 4]
```diff
@@ -1 +1,14 @@
-# chess
\ No newline at end of file
+# Java chess game
```
Comment (AlexisGlez) [inline README.md:1]: The Pull request name says that this was creating only tests, but you are adding Maven, Readmes, changing logic, etc. Remember to do only 1 thing per Pull Request

**Review by AlexisGlez — COMMENTED** [review]
Tests are looking pretty good 💯 

I only left some comments, also, remember that PR titles need to use imperative form of the verb:
```
added testing for most classes --> add tests for chess pieces
```

File: src/test/java/kevinglezgmz/chess/gameEngine/GameEngineTest.java (hunk 1 above)
Comment (kevinglezgmz) [reply src/test/java/kevinglezgmz/chess/gameEngine/GameEngineTest.java:11]: Forgot to remove the default test template given by maven.

**Pull request merged**